frontend/src/Components/Navbar.jsx

```
@@ -1,19 +1,34 @@
+        <Link to="/contributions" className={location.pathname === "/contributions" ? "active" : ""}>
+          Contributions
+        </Link>
+        <Link to="/membership" className={location.pathname === "/membership" ? "active" : ""}>
+          Membership
+        </Link>
+        <Link to="/contributors" className={location.pathname === "/contributors" ? "active" : ""}>
+          Contributors
+        </Link>
       </div>
+      <Link to="/login">
+        <button className="l-btn">Login</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)